webapp/components/interop-dashboard.js

```diff
@@ -456,6 +466,28 @@ class InteropDashboard extends PolymerElement {
       },
       dataManager: Object,
       scores: Object,
+      tableState: {
+        type: Object,
+        value: {
+          sortColumn: -1,
+          sortOrder: 'asc',
+          sorted: true
+        }
+      },
+      toggleStates: {
+        type: Array,
+        value: [
+          true,
+          false,
+          false,
+          false,
+          false
+        ]
+      },
```

Comment:
After testing this, it looks like this variable to control the arrow direction always has 1 or less value equal to true. I think you can do away with this array altogether if you change the logic of your `getSortIcon()` method to only display a downward arrow if the index is equal to the sort column and the sort order is ascending.